client/me/purchases/manage-purchase/index.jsx

```diff
@@ -636,6 +636,19 @@ class ManagePurchase extends Component {
 		return ! hasLoadedDomains;
 	}
 
+	getProductDisplayName() {
```

Comment:
This works, but is there a reason you're not making the change in /lib/purchases/index.js (maybe in [`getDisplayName()`](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/purchases/index.js#L112), or alternatively a new utility like `getDisplayNameWithTerm`)?

Reply:
@michaeldcain I didn't change `getDisplayName()` because the function is shared between the manage purchase page and the purchases list that is not included in the request. Let me check if @paulbonahora thinks we should add the "Monthly" label to the list items.

Reply:
The PR has been updated as per @michaeldcain 's suggestion. :)